py3status/modules/xrandr_toggle.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle monitor output from the status bar using xrandr.
+
```

Comment:
To me this isn't very clear.  What are we toggling?  Something like 'Toggles a monitor on or off' feels more explicit.

Also some more information would be nice.